### PR TITLE
Backport fix shared poller OOM to 0.13

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Detailed description**:
- Bump versions for 0.13.2
- Backport following fixes from #833
  - Add ability to set log levels without restart (via curl, not persistent across restarts)
  - Fix Java max heap parameter not taking effect, causing the OOM
  - Fix not crashing on OOM so the process can recover
  - Fix async logging not enabled
  - Fix JPA Criteria based `TopicMessageRepository.findByFilter()` using literals for primitives instead of bind parameters, exploding Hibernate's and JDBC's query plan cache
  - Fix Hibernate storing topic messages in its session unnecessarily by marking queries as read-only

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: hedera-mirror-grpc/scripts/hedera-mirror-grpc.service [hunk 1]
```diff
@@ -3,7 +3,8 @@ After=syslog.target
 Description=Hedera Mirror Node GRPC API
 
 [Service]
-ExecStart=/usr/bin/java -jar hedera-mirror-grpc.jar --spring.config.additional-location=file:/usr/etc/hedera-mirror-grpc/ -DLog4jContextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector -Xmx10g
+ExecStart=/usr/bin/java -Xms5g -Xmx12g -XX:+CrashOnOutOfMemoryError -DLog4jContextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector -jar hedera-mirror-grpc.jar --spring.config.additional-location=file:/usr/etc/hedera-mirror-grpc/
```
Comment (Nana-EC) [inline hedera-mirror-grpc/scripts/hedera-mirror-grpc.service:6]: We should probably add documentation for this in the Operations.md

Comment (Nana-EC) [inline hedera-mirror-grpc/scripts/hedera-mirror-grpc.service:6]: We should probably add documentation for this in the Operations.md

File: hedera-mirror-importer/scripts/hedera-mirror-importer.service [hunk 2]
```diff
@@ -3,8 +3,7 @@ After=syslog.target
 Description=Hedera Mirror Importer
 
 [Service]
-ExecStart=/usr/bin/java -jar hedera-mirror-importer.jar -Djavax.net.ssl.trustStorePassword=changeit -DLog4jContextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector --spring.config.additional-location=file:/usr/etc/hedera-mirror-importer/
-
+ExecStart=/usr/bin/java -Xms2g -Xmx10g -XX:+CrashOnOutOfMemoryError -DLog4jContextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector -jar hedera-mirror-importer.jar --spring.config.additional-location=file:/usr/etc/hedera-mirror-importer/
```
Comment (Nana-EC) [inline hedera-mirror-importer/scripts/hedera-mirror-importer.service:6]: Also add documentation for these config values in the Operations.md

File: hedera-mirror-grpc/scripts/hedera-mirror-grpc.service (hunk 1 above)
Comment (steven-sheehy) [reply hedera-mirror-grpc/scripts/hedera-mirror-grpc.service:6]: We can, but these are not really suggested memory settings for customers. They're more like "we have all this extra memory on the VM let's use it" settings.

**Pull request merged**